src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    White,
+    black,
+    yellow,
+    blue,
+    orange,
+    green
```

Comment:
Enum values should be in uppercase letters. According to the Java naming conventions, enum constants should be in uppercase letters. For example, `WHITE` instead of `White`, and `BLACK` instead of `black`. Please adjust all enum values to follow this convention.